kod/object/item/passitem/pforget/forgetxwi.kod

```diff
@@ -0,0 +1,76 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+ForgetPotionXaerdunWitchery is ForgetPotion
+
+constants:
+
+   include blakston.khd
+
+resources:
+
+   forgetpotionXaerdunWitchery_desc_rsc = \
+      "Quaffing this elixir will break your alliance with Xaerdun, "
+      "and erase any knowledge of his granted powers from your mind."
+
+classvars:
+
+   viForget_school = SS_WITCHERY
+   viForget_subschool = SS_WITCHERY_XAERDUN
+   vrDesc = forgetpotionXaerdunWitchery_desc_rsc
+
+properties:
+
+messages:
+
+   QuaffTimer()
+   {
+      local i, lSpells, oSpell, iSpellnum;
+
+      ptQuaff = $;
+      lSpells = Send(poApply_target,@GetSpellList);
+      piCount = 0;
+      
+      foreach i in lSpells
+      {
+         iSpellNum = send(poApply_target,@DecodeSpellNum,#compound=i);
+         oSpell = send(SYS,@FindSpellByNum,#num=iSpellNum);
+
+         if Send(oSpell,@GetSchool) = viForget_school
+
+            %%% ??? whats this?
+            %AND Send(oSpell,@GetSubSchool) = viForget_subschool
```

Comment:
Witchery has / will have subschools. 

If you're trying to merge the big desert thing, you should let me take a look at it and rebase it. I bet it's out of date after ~6 months gathering dust.